website/src/docs/uppy.md

```diff
@@ -767,6 +767,22 @@ uppy.on('upload', (data) => {
 })
 ```
 
+### `progress`
+
+Fired each time the total upload progress is updated:
+
+**Parameters**
+- `progress` - An integer representing the total upload progress.
```

Comment:
```suggestion
- `progress` - An integer (0-100) representing the total upload progress.
```
Think it'd be best to stress here that this is 0-100 and not 0.0-1.0, because it's a little unusual.